node/src/Commands.ts

```diff
@@ -2800,3 +2800,48 @@ export function createLCS(
 
     return createCommand(RequestType.LCS, args);
 }
+
+/**
+ * This base class represents the common set of optional arguments for the SCAN family of commands.
+ * Concrete implementations of this class are tied to specific SCAN commands (SCAN, HSCAN, SSCAN,
+ * and ZSCAN).
+ */
+export type BaseScanOptions = {
```

Comment:
```suggestion
export type ZScanOptions = BaseScanOptions;

type BaseScanOptions = {
```

Reply:
Based on our discussion. We are keeping it as is and add anonymous types for other scan commands. So it would be something similar to `options?: BaseScanOptions & { otherOption: string }`.